src/preact.d.ts

```diff
@@ -141,6 +147,7 @@ declare namespace JSX {
 		y1?:number | string;
 		y2?:number | string;
 		y?:number | string;
+		[propName: string]: any;
```

Comment:
Hrmm yeah, I added it because of this error I saw:

> Object literal may only specify known properties, and ''data-canonical'' does not exist in type 'HTMLAttributes & SVGAttributes'